journald/lib_journald.hpp

```diff
@@ -18,10 +18,105 @@ namespace journald {
 
 class JournaldContainerLoggerProcess;
 
-struct Flags : public virtual flags::FlagsBase
+
+// These flags are loaded twice: once when the `ContainerLogger` module
+// is created and each time before launching executors. The flags loaded
+// at module creation act as global default values, whereas flags loaded
+// prior to executors can override the global values.
+struct LoggerFlags : public virtual flags::FlagsBase
+{
+  LoggerFlags()
+  {
+    add(&LoggerFlags::destination_type,
+        "destination_type",
+        "Determines where logs should be piped.\n"
+        "Valid destinations include: 'journald', 'sandbox', or 'both'.",
+        "journald",
```

Comment:
I used different defaults because this module's original intent was for the `journald` mode.  Whereas the DC/OS PR explicitly wanted to have both options for the moment.